x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_actions/actions/component/index_item.tsx

```diff
@@ -18,28 +18,29 @@ import {
 } from '@elastic/eui';
 import React from 'react';
 import { ENTERPRISE_SEARCH_CONTENT_APP_ID } from '@kbn/deeplinks-search';
-import { MANAGEMENT_APP_ID } from '@kbn/deeplinks-management/constants';
+
+import { SEARCH_INDICES } from '@kbn/deeplinks-search/constants';
 
 import { useKibana } from '../../../../../../hooks/use_kibana';
 import { InferenceUsageInfo } from '../../../../types';
-import { PIPELINE_URL } from '../../../../constants';
+import { SERVERLESS_INDEX_MANAGEMENT_URL } from '../../../../constants';
 
 interface UsageProps {
   usageItem: InferenceUsageInfo;
 }
-export const UsageItem: React.FC<UsageProps> = ({ usageItem }) => {
+export const IndexItem: React.FC<UsageProps> = ({ usageItem }) => {
   const {
-    services: { application },
+    services: { application, serverless },
   } = useKibana();
-  const handleNavigateToIndex = () => {
-    if (usageItem.type === 'Index') {
-      application?.navigateToApp(ENTERPRISE_SEARCH_CONTENT_APP_ID, {
-        path: `search_indices/${usageItem.id}`,
+  const navigateToIndex = async () => {
+    if (serverless) {
```

Comment:
I'm ok with this for now but we should create an issue to introduce a locator for the index details page.